pkg/object/meshcontroller/api/api_ingress.go

```diff
@@ -73,34 +74,24 @@ func (a *API) createIngress(w http.ResponseWriter, r *http.Request) {
 	pbIngressSpec := &v1alpha1.Ingress{}
 	ingressSpec := &spec.Ingress{}
 
-	ingressName, err := a.readIngressName(w, r)
-	if err != nil {
-		api.HandleAPIError(w, r, http.StatusBadRequest, err)
-		return
-	}
-	err = a.readAPISpec(w, r, pbIngressSpec, ingressSpec)
+	err := a.readAPISpec(w, r, pbIngressSpec, ingressSpec)
 	if err != nil {
 		api.HandleAPIError(w, r, http.StatusBadRequest, err)
 		return
 	}
-	if ingressName != ingressSpec.Name {
```

Comment:
I think we should keep this name consistency checking since I can't find this logic in `readAPISpec`.

Reply:
Please check the detailed adjustment of the path. There is no name in the path anymore.

Reply:
Ok, after removing the `{ingressName}` in the RESTful API path, this is fine to me.